content/ux-guide/resources/assisting-in-research.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Assisting in research
+description: "How to assist in user research at 18F."
+permalink: /ux-guide/research/assist/
+layout: layouts/page
+tags: ux-guide
```

Comment:
I'm not sure how to do this one in Eleventy, since the `research` sidenav on the Jekyll page doesn't connect to this `assist` page at all. But I'm also leaning toward not worrying about it, since it doesn't really add any useful orientation information.

Reply:
Created ticket [#378](https://github.com/18F/TLC-crew/issues/378)